### PR TITLE
zenlib: add 9999, 2.2.0; ugrd: add 1.16.0

### PR DESCRIPTION
**Pull request by desultory**
Version bumps.

---

Please check all the boxes that apply:

- [ x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [ x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [ x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x ] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-python/pycpio/pycpio-9999.ebuild [hunk 1]
```diff
@@ -14,4 +14,4 @@ EGIT_REPO_URI="https://github.com/desultory/${PN}"
 LICENSE="GPL-2"
 SLOT="0"
 
-RDEPEND=">=dev-python/zenlib-2.1.2[${PYTHON_USEDEP}]"
+RDEPEND=">=dev-python/zenlib-9999[${PYTHON_USEDEP}]"
```
Comment (AndrewAmmerlaan) [inline dev-python/pycpio/pycpio-9999.ebuild:17]: This should be two separate commits, otherwise LGTM.

**Comment by AndrewAmmerlaan** [comment]
> **dev-python/pycpio**: ~dev[at]pyl.onl~, @AndrewAmmerlaan
> **dev-python/zenlib**: ~dev[at]pyl.onl~, @AndrewAmmerlaan
> **sys-kernel/ugrd**: ~dev[at]pyl.onl~, @AndrewAmmerlaan

To have the bot automatically tag you on Pull Requests touching your packages, please register this e-mail address (as a secondary e-mail address) in GitHub.

File: dev-python/zenlib/zenlib-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PYTHON_COMPAT=( python3_{11..12} )
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Useful python decorators and utilities"
+HOMEPAGE="https://github.com/desultory/zenlib"
+EGIT_REPO_URI="https://github.com/desultory/zenlib.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
```
Comment (AndrewAmmerlaan) [inline dev-python/zenlib/zenlib-9999.ebuild:16]: And keywords should be dropped from the live ebuild

**Comment by desultory** [comment]
> > **dev-python/pycpio**: ~dev[at]pyl.onl~, @AndrewAmmerlaan
> > **dev-python/zenlib**: ~dev[at]pyl.onl~, @AndrewAmmerlaan
> > **sys-kernel/ugrd**: ~dev[at]pyl.onl~, @AndrewAmmerlaan
> 
> To have the bot automatically tag you on Pull Requests touching your packages, please register this e-mail address (as a secondary e-mail address) in GitHub.

hmmm that email is registered, not sure what the issue is:
image

**Comment by AndrewAmmerlaan** [comment]
> hmmm that email is registered, not sure what the issue is:

I suppose the issue is that it is not visible to the bot, it might have to be public, not completely sure.

**Comment by desultory** [comment]
> > hmmm that email is registered, not sure what the issue is:
> 
> I suppose the issue is that it is not visible to the bot, it might have to be public, not completely sure.

maybe I should update my commits to use that github email?

**Comment by AndrewAmmerlaan** [comment]
> maybe I should update my commits to use that github email?

The author of the commit doesn't matter I think, it gets the e-mail address from `metadata.xml` and tries to find the GitHub user that belongs to it.

File: dev-python/zenlib/zenlib-9999.ebuild [hunk 3]
```diff
@@ -13,4 +13,3 @@ EGIT_REPO_URI="https://github.com/desultory/zenlib.git"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm64"
```
Comment (AndrewAmmerlaan) [inline dev-python/zenlib/zenlib-9999.ebuild]: Please squash this commit into the first commit.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:
